src/views/clusteroverview/MonitoringTab/migrations-card/components/MigrationsLimitionsPopover/hooks/useHCMigrations.ts

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react';
+
+import { V1MigrationConfiguration } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { HyperConvergedModelGroupVersionKind } from '@kubevirt-utils/models';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+
+import { HyperConverged } from '../../../../../utils/types';
+import { getHyperConvergedObject } from '../../../../../utils/utils';
+
+const useHCMigrations = (): V1MigrationConfiguration => {
+  const [hyperConvergeData] = useK8sWatchResource<HyperConverged[]>({
+    groupVersionKind: HyperConvergedModelGroupVersionKind,
+    isList: true,
+  });
+
+  const hyperConverge = useMemo(
+    () => getHyperConvergedObject(hyperConvergeData),
```

Comment:
Should get... Be exported to general util?

Reply:
I guess that is not a bad idea, do you think we should apply it to this PR or in a seprated one?

Reply:
Whatever you prefer